students/dave_arasim/session02/fizzbuzz.py

```diff
@@ -0,0 +1,12 @@
+#Fizz Buzz Exercise for Lesson 2
+
+for x in range(100):
+    x += 1 #changes range 0-99 to 1-100
```

Comment:
This is not right. You'll get 1, 3, 5, 7, ... for x, not 1,2,3,4,5, ... .

To get 1, 2, 3, ..., 100, this line should be removed and the `range(100)` should be changed to `range(1, 101)`. Search the web for `python range` and study how it's used.